src/Factory.php

```diff
@@ -204,47 +203,47 @@ public static function toFile($filename, $config)
     /**
      * Set reader plugin manager
      *
-     * @param ReaderPluginManager $readers
+     * @param ContainerInterface $readers
      * @return void
      */
-    public static function setReaderPluginManager(ReaderPluginManager $readers)
+    public static function setReaderPluginManager(ContainerInterface $readers)
     {
         static::$readers = $readers;
     }
 
     /**
      * Get the reader plugin manager
      *
-     * @return ReaderPluginManager
+     * @return ContainerInterface
```

Comment:
The whole point with this change was to signal that you can provide generic implementations based on `ContainerInterface`.

I'll update the method description to indicate the default implementation, however.